docs/swagger-doc-coverage.md

```diff
@@ -0,0 +1,178 @@
+# Adding REST endpoints to Swagger UI
+
+[Swagger UI](http://swagger.io/swagger-ui/) is a graphical front-end that displays
+REST API documentation with working examples. It uses a description of the API
+defined according to the Swagger/OpenAPI specification. For Zulip, this is in
+`/static/yaml/zulip.yaml`, and the generated API reference is available at a URL
+similar to other Zulip documentation. You can find an example at
+[/apidocs](https://zulipchat.com/apidocs/). This page is a basic introduction to the
```

Comment:
Maybe instead suggest the location in development, e.g. `http://localhost:9991/apidocs/`?  That's what they'll want to look at in order to iterate on it in development.